wasm2c/wasm-rt-impl.c

```diff
@@ -345,7 +349,8 @@ void wasm_rt_allocate_memory(wasm_rt_memory_t* memory,
 #endif
 }
 
-uint64_t wasm_rt_grow_memory(wasm_rt_memory_t* memory, uint64_t delta) {
+static uint64_t wasm_rt_grow_memory_helper(wasm_rt_memory_t* memory,
```

Comment:
How about just calling this `grow_memory`.. or `grow_memory_impl`, no need to namespace/prefix it since its static.

Reply:
Will fix